sensor_msgs/Image image
geometry_msgs/Point[] landmarks
---
float32[]             eyes_closed
